docker/Dockerfile

```diff
@@ -44,9 +44,10 @@ RUN \
     # Install rapidjson from source
     curl -kOL https://github.com/miloyip/rapidjson/archive/v1.0.2.tar.gz && \
     tar xfz v1.0.2.tar.gz && \
-    mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include && \
+    mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include
+RUN \
     # Install orion from source
-    git clone https://github.com/telefonicaid/fiware-orion && \
+    git clone https://github.com/giovannicuriel/fiware-orion && \
```

Comment:
I'm also sure that this won't be accepted. But for docker hub build the correct image, I needed to change this. Wouldn't it be better to copy the code from the current folder to the image? Or at least have alternate dockerfiles - one for local code (so we can generate an image without pushing code) and other for the original code from github.